 Network rinha2024  Creating
 Network rinha2024  Created
 Container paulohenriqu-db-1  Creating
 Container paulohenriqu-db-1  Created
 Container paulohenriqu-api01-1  Creating
 Container paulohenriqu-api02-1  Creating
 Container paulohenriqu-api01-1  Created
 Container paulohenriqu-api02-1  Created
 Container paulohenriqu-nginx-1  Creating
 Container paulohenriqu-nginx-1  Created
Attaching to api01-1, api02-1, db-1, nginx-1
db-1     | The files belonging to this database system will be owned by user "postgres".
db-1     | This user must also own the server process.
db-1     | 
db-1     | The database cluster will be initialized with locale "en_US.utf8".
db-1     | The default database encoding has accordingly been set to "UTF8".
db-1     | The default text search configuration will be set to "english".
db-1     | 
db-1     | Data page checksums are disabled.
db-1     | 
db-1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1     | creating subdirectories ... ok
db-1     | selecting dynamic shared memory implementation ... posix
db-1     | selecting default max_connections ... 100
db-1     | selecting default shared_buffers ... 128MB
api02-1  | 
api02-1  |   .   ____          _            __ _ _
api02-1  |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
api02-1  | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
api02-1  |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
api02-1  |   '  |____| .__|_| |_|_| |_\__, | / / / /
api02-1  |  =========|_|==============|___/=/_/_/_/
api02-1  |  :: Spring Boot ::                (v3.2.2)
api02-1  | 
api02-1  | 2024-02-14T05:29:27.200Z  INFO 1 --- [           main] com.phs.rinha.RinhaApplication           : Starting AOT-processed RinhaApplication using Java 21.0.2 with PID 1 (/rinha started by root in /)
api02-1  | 2024-02-14T05:29:27.200Z  INFO 1 --- [           main] com.phs.rinha.RinhaApplication           : No active profile set, falling back to 1 default profile: "default"
db-1     | selecting default time zone ... Etc/UTC
db-1     | creating configuration files ... ok
api01-1  | 
api01-1  |   .   ____          _            __ _ _
api01-1  |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
api01-1  | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
api01-1  |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
api01-1  |   '  |____| .__|_| |_|_| |_\__, | / / / /
api01-1  |  =========|_|==============|___/=/_/_/_/
api01-1  |  :: Spring Boot ::                (v3.2.2)
api01-1  | 
api01-1  | 2024-02-14T05:29:27.299Z  INFO 1 --- [           main] com.phs.rinha.RinhaApplication           : Starting AOT-processed RinhaApplication using Java 21.0.2 with PID 1 (/rinha started by root in /)
api01-1  | 2024-02-14T05:29:27.299Z  INFO 1 --- [           main] com.phs.rinha.RinhaApplication           : No active profile set, falling back to 1 default profile: "default"
api02-1  | 2024-02-14T05:29:27.547Z  INFO 1 --- [           main] o.s.b.web.embedded.netty.NettyWebServer  : Netty started on port 8080
api02-1  | 2024-02-14T05:29:27.547Z  INFO 1 --- [           main] com.phs.rinha.RinhaApplication           : Started RinhaApplication in 0.374 seconds (process running for 0.38)
api01-1  | 2024-02-14T05:29:27.664Z  INFO 1 --- [           main] o.s.b.web.embedded.netty.NettyWebServer  : Netty started on port 8080
api01-1  | 2024-02-14T05:29:27.665Z  INFO 1 --- [           main] com.phs.rinha.RinhaApplication           : Started RinhaApplication in 0.426 seconds (process running for 0.431)
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
db-1     | running bootstrap script ... ok
api01-1  | 2024-02-14T05:29:30.112Z ERROR 1 --- [tor-tcp-epoll-2] a.w.r.e.AbstractErrorWebExceptionHandler : [22721238-1]  500 Server Error for HTTP GET "/clientes/1/extrato"
api01-1  | 
api01-1  | org.springframework.dao.DataAccessResourceFailureException: Failed to obtain R2DBC Connection
api01-1  | 	at org.springframework.r2dbc.connection.ConnectionFactoryUtils.lambda$getConnection$0(ConnectionFactoryUtils.java:100) ~[na:na]
api01-1  | 	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
api01-1  | Error has been observed at the following site(s):
api01-1  | 	*__checkpoint ⇢ Handler com.phs.rinha.adapter.in.ClientController#getExtrato(int) [DispatcherHandler]
api01-1  | 	*__checkpoint ⇢ HTTP GET "/clientes/1/extrato" [ExceptionHandlingWebHandler]
api01-1  | Original Stack Trace:
api01-1  | 		at org.springframework.r2dbc.connection.ConnectionFactoryUtils.lambda$getConnection$0(ConnectionFactoryUtils.java:100) ~[na:na]
api01-1  | 		at reactor.core.publisher.Mono.lambda$onErrorMap$28(Mono.java:3799) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:94) ~[na:na]
api01-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106) ~[na:na]
api01-1  | 		at reactor.core.publisher.FluxRetry$RetrySubscriber.onError(FluxRetry.java:96) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180) ~[rinha:3.6.2]
api01-1  | 		at reactor.pool.AbstractPool$Borrower.fail(AbstractPool.java:488) ~[rinha:1.0.5]
api01-1  | 		at reactor.pool.SimpleDequePool.lambda$drainLoop$9(SimpleDequePool.java:436) ~[rinha:1.0.5]
api01-1  | 		at reactor.core.publisher.FluxDoOnEach$DoOnEachSubscriber.onError(FluxDoOnEach.java:186) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.MonoSubscribeOn$SubscribeOnSubscriber.onError(MonoSubscribeOn.java:152) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onError(FluxMapFuseable.java:142) ~[na:na]
api01-1  | 		at reactor.core.publisher.FluxContextWrite$ContextWriteSubscriber.onError(FluxContextWrite.java:121) ~[na:na]
api01-1  | 		at reactor.core.publisher.MonoSingle$SingleSubscriber.onError(MonoSingle.java:150) ~[na:na]
api01-1  | 		at io.r2dbc.postgresql.util.FluxDiscardOnCancel$FluxDiscardOnCancelSubscriber.onError(FluxDiscardOnCancel.java:97) ~[na:na]
api01-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106) ~[na:na]
api01-1  | 		at reactor.core.publisher.Operators.error(Operators.java:198) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.MonoError.subscribe(MonoError.java:53) ~[na:na]
api01-1  | 		at reactor.core.publisher.Mono.subscribe(Mono.java:4512) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103) ~[na:na]
api01-1  | 		at reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.MonoDelayUntil$DelayUntilCoordinator.onError(MonoDelayUntil.java:201) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onError(FluxMapFuseable.java:142) ~[na:na]
api01-1  | 		at reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:205) ~[rinha:3.6.2]
api01-1  | 		at reactor.netty.resources.NewConnectionProvider$DisposableConnect.onError(NewConnectionProvider.java:156) ~[na:na]
api01-1  | 		at reactor.core.publisher.MonoFlatMap$FlatMapMain.secondError(MonoFlatMap.java:241) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.MonoFlatMap$FlatMapInner.onError(MonoFlatMap.java:315) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106) ~[na:na]
api01-1  | 		at reactor.core.publisher.Operators.error(Operators.java:198) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.MonoError.subscribe(MonoError.java:53) ~[na:na]
api01-1  | 		at reactor.core.publisher.Mono.subscribe(Mono.java:4512) ~[rinha:3.6.2]
api01-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103) ~[na:na]
api01-1  | 		at reactor.netty.transport.TransportConnector$MonoChannelPromise.tryFailure(TransportConnector.java:576) ~[rinha:1.1.15]
api01-1  | 		at reactor.netty.transport.TransportConnector$MonoChannelPromise.setFailure(TransportConnector.java:522) ~[rinha:1.1.15]
api01-1  | 		at reactor.netty.transport.TransportConnector.lambda$doConnect$7(TransportConnector.java:261) ~[na:na]
api01-1  | 		at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:590) ~[rinha:4.1.105.Final]
api01-1  | 		at io.netty.util.concurrent.DefaultPromise.notifyListeners0(DefaultPromise.java:583) ~[rinha:4.1.105.Final]
api01-1  | 		at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:559) ~[rinha:4.1.105.Final]
api01-1  | 		at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:492) ~[rinha:4.1.105.Final]
api01-1  | 		at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:636) ~[rinha:4.1.105.Final]
api01-1  | 		at io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:629) ~[rinha:4.1.105.Final]
api01-1  | 		at io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:118) ~[rinha:4.1.105.Final]
api01-1  | 		at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.fulfillConnectPromise(AbstractEpollChannel.java:675) ~[rinha:4.1.105.Final]
api01-1  | 		at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:694) ~[rinha:4.1.105.Final]
api01-1  | 		at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:567) ~[rinha:4.1.105.Final]
api01-1  | 		at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:499) ~[na:na]
api01-1  | 		at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:407) ~[na:na]
api01-1  | 		at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[na:na]
api01-1  | 		at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[na:na]
api01-1  | 		at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[na:na]
api01-1  | 		at java.base@21.0.2/java.lang.Thread.runWith(Thread.java:1596) ~[rinha:na]
api01-1  | 		at java.base@21.0.2/java.lang.Thread.run(Thread.java:1583) ~[rinha:na]
api01-1  | 		at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:833) ~[rinha:na]
api01-1  | 		at org.graalvm.nativeimage.builder/com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:211) ~[na:na]
api01-1  | Caused by: io.r2dbc.postgresql.PostgresqlConnectionFactory$PostgresConnectionException: Cannot connect to db/<unresolved>:5432
api01-1  | 	at io.r2dbc.postgresql.PostgresqlConnectionFactory.cannotConnect(PostgresqlConnectionFactory.java:188) ~[rinha:1.0.4.RELEASE]
api01-1  | 	at io.r2dbc.postgresql.PostgresqlConnectionFactory.lambda$doCreateConnection$7(PostgresqlConnectionFactory.java:153) ~[rinha:1.0.4.RELEASE]
api01-1  | 	at reactor.core.publisher.Mono.lambda$onErrorMap$28(Mono.java:3799) ~[rinha:3.6.2]
api01-1  | 	at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:94) ~[na:na]
api01-1  | 	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180) ~[rinha:3.6.2]
api01-1  | 	at reactor.core.publisher.MonoDelayUntil$DelayUntilCoordinator.onError(MonoDelayUntil.java:201) ~[rinha:3.6.2]
api01-1  | 	at reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onError(FluxMapFuseable.java:142) ~[na:na]
api01-1  | 	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180) ~[rinha:3.6.2]
api01-1  | 	at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:205) ~[rinha:3.6.2]
api01-1  | 	at reactor.netty.resources.NewConnectionProvider$DisposableConnect.onError(NewConnectionProvider.java:156) ~[na:na]
api01-1  | 	at reactor.core.publisher.MonoFlatMap$FlatMapMain.secondError(MonoFlatMap.java:241) ~[rinha:3.6.2]
api01-1  | 	at reactor.core.publisher.MonoFlatMap$FlatMapInner.onError(MonoFlatMap.java:315) ~[rinha:3.6.2]
api01-1  | 	at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106) ~[na:na]
api01-1  | 	at reactor.core.publisher.Operators.error(Operators.java:198) ~[rinha:3.6.2]
api01-1  | 	at reactor.core.publisher.MonoError.subscribe(MonoError.java:53) ~[na:na]
api01-1  | 	at reactor.core.publisher.Mono.subscribe(Mono.java:4512) ~[rinha:3.6.2]
api01-1  | 	at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103) ~[na:na]
api01-1  | 	at reactor.netty.transport.TransportConnector$MonoChannelPromise.tryFailure(TransportConnector.java:576) ~[rinha:1.1.15]
api01-1  | 	at reactor.netty.transport.TransportConnector$MonoChannelPromise.setFailure(TransportConnector.java:522) ~[rinha:1.1.15]
api01-1  | 	at reactor.netty.transport.TransportConnector.lambda$doConnect$7(TransportConnector.java:261) ~[na:na]
api01-1  | 	at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:590) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.util.concurrent.DefaultPromise.notifyListeners0(DefaultPromise.java:583) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:559) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:492) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:636) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:629) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:118) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.fulfillConnectPromise(AbstractEpollChannel.java:675) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:694) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:567) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:499) ~[na:na]
api01-1  | 	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:407) ~[na:na]
api01-1  | 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[na:na]
api01-1  | 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[na:na]
api01-1  | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[na:na]
api01-1  | 	at java.base@21.0.2/java.lang.Thread.runWith(Thread.java:1596) ~[rinha:na]
api01-1  | 	at java.base@21.0.2/java.lang.Thread.run(Thread.java:1583) ~[rinha:na]
api01-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:833) ~[rinha:na]
api01-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:211) ~[na:na]
api01-1  | Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: db/192.168.128.2:5432
api01-1  | Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
api01-1  | 	at io.netty.channel.unix.Errors.newConnectException0(Errors.java:166) ~[na:na]
api01-1  | 	at io.netty.channel.unix.Errors.handleConnectErrno(Errors.java:131) ~[na:na]
api01-1  | 	at io.netty.channel.unix.Socket.finishConnect(Socket.java:359) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:711) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:688) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:567) ~[rinha:4.1.105.Final]
api01-1  | 	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:499) ~[na:na]
api01-1  | 	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:407) ~[na:na]
api01-1  | 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[na:na]
api01-1  | 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[na:na]
api01-1  | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[na:na]
api01-1  | 	at java.base@21.0.2/java.lang.Thread.runWith(Thread.java:1596) ~[rinha:na]
api01-1  | 	at java.base@21.0.2/java.lang.Thread.run(Thread.java:1583) ~[rinha:na]
api01-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:833) ~[rinha:na]
api01-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:211) ~[na:na]
api01-1  | 
db-1     | performing post-bootstrap initialization ... ok
db-1     | initdb: warning: enabling "trust" authentication for local connections
db-1     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1     | syncing data to disk ... ok
db-1     | 
db-1     | 
db-1     | Success. You can now start the database server using:
db-1     | 
db-1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1     | 
db-1     | waiting for server to start....2024-02-14 05:29:31.598 UTC [48] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-14 05:29:31.598 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-14 05:29:31.601 UTC [51] LOG:  database system was shut down at 2024-02-14 05:29:30 UTC
db-1     | 2024-02-14 05:29:31.606 UTC [48] LOG:  database system is ready to accept connections
db-1     |  done
db-1     | server started
api02-1  | 2024-02-14T05:29:32.136Z ERROR 1 --- [tor-tcp-epoll-2] a.w.r.e.AbstractErrorWebExceptionHandler : [cbeed732-1]  500 Server Error for HTTP GET "/clientes/1/extrato"
api02-1  | 
api02-1  | org.springframework.dao.DataAccessResourceFailureException: Failed to obtain R2DBC Connection
api02-1  | 	at org.springframework.r2dbc.connection.ConnectionFactoryUtils.lambda$getConnection$0(ConnectionFactoryUtils.java:100) ~[na:na]
api02-1  | 	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
api02-1  | Error has been observed at the following site(s):
api02-1  | 	*__checkpoint ⇢ Handler com.phs.rinha.adapter.in.ClientController#getExtrato(int) [DispatcherHandler]
api02-1  | 	*__checkpoint ⇢ HTTP GET "/clientes/1/extrato" [ExceptionHandlingWebHandler]
api02-1  | Original Stack Trace:
api02-1  | 		at org.springframework.r2dbc.connection.ConnectionFactoryUtils.lambda$getConnection$0(ConnectionFactoryUtils.java:100) ~[na:na]
api02-1  | 		at reactor.core.publisher.Mono.lambda$onErrorMap$28(Mono.java:3799) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:94) ~[na:na]
api02-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106) ~[na:na]
api02-1  | 		at reactor.core.publisher.FluxRetry$RetrySubscriber.onError(FluxRetry.java:96) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180) ~[rinha:3.6.2]
api02-1  | 		at reactor.pool.AbstractPool$Borrower.fail(AbstractPool.java:488) ~[rinha:1.0.5]
api02-1  | 		at reactor.pool.SimpleDequePool.lambda$drainLoop$9(SimpleDequePool.java:436) ~[rinha:1.0.5]
api02-1  | 		at reactor.core.publisher.FluxDoOnEach$DoOnEachSubscriber.onError(FluxDoOnEach.java:186) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.MonoSubscribeOn$SubscribeOnSubscriber.onError(MonoSubscribeOn.java:152) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onError(FluxMapFuseable.java:142) ~[na:na]
api02-1  | 		at reactor.core.publisher.FluxContextWrite$ContextWriteSubscriber.onError(FluxContextWrite.java:121) ~[na:na]
api02-1  | 		at reactor.core.publisher.MonoSingle$SingleSubscriber.onError(MonoSingle.java:150) ~[na:na]
api02-1  | 		at io.r2dbc.postgresql.util.FluxDiscardOnCancel$FluxDiscardOnCancelSubscriber.onError(FluxDiscardOnCancel.java:97) ~[na:na]
api02-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106) ~[na:na]
api02-1  | 		at reactor.core.publisher.Operators.error(Operators.java:198) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.MonoError.subscribe(MonoError.java:53) ~[na:na]
api02-1  | 		at reactor.core.publisher.Mono.subscribe(Mono.java:4512) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103) ~[na:na]
api02-1  | 		at reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.MonoDelayUntil$DelayUntilCoordinator.onError(MonoDelayUntil.java:201) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onError(FluxMapFuseable.java:142) ~[na:na]
api02-1  | 		at reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:205) ~[rinha:3.6.2]
api02-1  | 		at reactor.netty.resources.NewConnectionProvider$DisposableConnect.onError(NewConnectionProvider.java:156) ~[na:na]
api02-1  | 		at reactor.core.publisher.MonoFlatMap$FlatMapMain.secondError(MonoFlatMap.java:241) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.MonoFlatMap$FlatMapInner.onError(MonoFlatMap.java:315) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106) ~[na:na]
api02-1  | 		at reactor.core.publisher.Operators.error(Operators.java:198) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.MonoError.subscribe(MonoError.java:53) ~[na:na]
api02-1  | 		at reactor.core.publisher.Mono.subscribe(Mono.java:4512) ~[rinha:3.6.2]
api02-1  | 		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103) ~[na:na]
api02-1  | 		at reactor.netty.transport.TransportConnector$MonoChannelPromise.tryFailure(TransportConnector.java:576) ~[rinha:1.1.15]
api02-1  | 		at reactor.netty.transport.TransportConnector$MonoChannelPromise.setFailure(TransportConnector.java:522) ~[rinha:1.1.15]
api02-1  | 		at reactor.netty.transport.TransportConnector.lambda$doConnect$7(TransportConnector.java:261) ~[na:na]
api02-1  | 		at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:590) ~[rinha:4.1.105.Final]
api02-1  | 		at io.netty.util.concurrent.DefaultPromise.notifyListeners0(DefaultPromise.java:583) ~[rinha:4.1.105.Final]
api02-1  | 		at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:559) ~[rinha:4.1.105.Final]
api02-1  | 		at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:492) ~[rinha:4.1.105.Final]
api02-1  | 		at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:636) ~[rinha:4.1.105.Final]
api02-1  | 		at io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:629) ~[rinha:4.1.105.Final]
api02-1  | 		at io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:118) ~[rinha:4.1.105.Final]
api02-1  | 		at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.fulfillConnectPromise(AbstractEpollChannel.java:675) ~[rinha:4.1.105.Final]
api02-1  | 		at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:694) ~[rinha:4.1.105.Final]
api02-1  | 		at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:567) ~[rinha:4.1.105.Final]
api02-1  | 		at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:499) ~[na:na]
api02-1  | 		at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:407) ~[na:na]
api02-1  | 		at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[na:na]
api02-1  | 		at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[na:na]
api02-1  | 		at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[na:na]
api02-1  | 		at java.base@21.0.2/java.lang.Thread.runWith(Thread.java:1596) ~[rinha:na]
api02-1  | 		at java.base@21.0.2/java.lang.Thread.run(Thread.java:1583) ~[rinha:na]
api02-1  | 		at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:833) ~[rinha:na]
api02-1  | 		at org.graalvm.nativeimage.builder/com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:211) ~[na:na]
api02-1  | Caused by: io.r2dbc.postgresql.PostgresqlConnectionFactory$PostgresConnectionException: Cannot connect to db/<unresolved>:5432
api02-1  | 	at io.r2dbc.postgresql.PostgresqlConnectionFactory.cannotConnect(PostgresqlConnectionFactory.java:188) ~[rinha:1.0.4.RELEASE]
api02-1  | 	at io.r2dbc.postgresql.PostgresqlConnectionFactory.lambda$doCreateConnection$7(PostgresqlConnectionFactory.java:153) ~[rinha:1.0.4.RELEASE]
api02-1  | 	at reactor.core.publisher.Mono.lambda$onErrorMap$28(Mono.java:3799) ~[rinha:3.6.2]
api02-1  | 	at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:94) ~[na:na]
api02-1  | 	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180) ~[rinha:3.6.2]
api02-1  | 	at reactor.core.publisher.MonoDelayUntil$DelayUntilCoordinator.onError(MonoDelayUntil.java:201) ~[rinha:3.6.2]
api02-1  | 	at reactor.core.publisher.FluxMapFuseable$MapFuseableSubscriber.onError(FluxMapFuseable.java:142) ~[na:na]
api02-1  | 	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onError(MonoFlatMap.java:180) ~[rinha:3.6.2]
api02-1  | 	at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:205) ~[rinha:3.6.2]
api02-1  | 	at reactor.netty.resources.NewConnectionProvider$DisposableConnect.onError(NewConnectionProvider.java:156) ~[na:na]
api02-1  | 	at reactor.core.publisher.MonoFlatMap$FlatMapMain.secondError(MonoFlatMap.java:241) ~[rinha:3.6.2]
api02-1  | 	at reactor.core.publisher.MonoFlatMap$FlatMapInner.onError(MonoFlatMap.java:315) ~[rinha:3.6.2]
api02-1  | 	at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:106) ~[na:na]
api02-1  | 	at reactor.core.publisher.Operators.error(Operators.java:198) ~[rinha:3.6.2]
api02-1  | 	at reactor.core.publisher.MonoError.subscribe(MonoError.java:53) ~[na:na]
api02-1  | 	at reactor.core.publisher.Mono.subscribe(Mono.java:4512) ~[rinha:3.6.2]
api02-1  | 	at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103) ~[na:na]
api02-1  | 	at reactor.netty.transport.TransportConnector$MonoChannelPromise.tryFailure(TransportConnector.java:576) ~[rinha:1.1.15]
api02-1  | 	at reactor.netty.transport.TransportConnector$MonoChannelPromise.setFailure(TransportConnector.java:522) ~[rinha:1.1.15]
api02-1  | 	at reactor.netty.transport.TransportConnector.lambda$doConnect$7(TransportConnector.java:261) ~[na:na]
api02-1  | 	at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:590) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.util.concurrent.DefaultPromise.notifyListeners0(DefaultPromise.java:583) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:559) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:492) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:636) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:629) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:118) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.fulfillConnectPromise(AbstractEpollChannel.java:675) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:694) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:567) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:499) ~[na:na]
api02-1  | 	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:407) ~[na:na]
api02-1  | 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[na:na]
api02-1  | 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[na:na]
api02-1  | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[na:na]
api02-1  | 	at java.base@21.0.2/java.lang.Thread.runWith(Thread.java:1596) ~[rinha:na]
api02-1  | 	at java.base@21.0.2/java.lang.Thread.run(Thread.java:1583) ~[rinha:na]
api02-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:833) ~[rinha:na]
api02-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:211) ~[na:na]
api02-1  | Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: db/192.168.128.2:5432
api02-1  | Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
api02-1  | 	at io.netty.channel.unix.Errors.newConnectException0(Errors.java:166) ~[na:na]
api02-1  | 	at io.netty.channel.unix.Errors.handleConnectErrno(Errors.java:131) ~[na:na]
api02-1  | 	at io.netty.channel.unix.Socket.finishConnect(Socket.java:359) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:711) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:688) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:567) ~[rinha:4.1.105.Final]
api02-1  | 	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:499) ~[na:na]
api02-1  | 	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:407) ~[na:na]
api02-1  | 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[na:na]
api02-1  | 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[na:na]
api02-1  | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[na:na]
api02-1  | 	at java.base@21.0.2/java.lang.Thread.runWith(Thread.java:1596) ~[rinha:na]
api02-1  | 	at java.base@21.0.2/java.lang.Thread.run(Thread.java:1583) ~[rinha:na]
api02-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:833) ~[rinha:na]
api02-1  | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:211) ~[na:na]
api02-1  | 
db-1     | CREATE DATABASE
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | CREATE INDEX
db-1     | INSERT 0 5
db-1     | 
db-1     | 
db-1     | 2024-02-14 05:29:32.407 UTC [48] LOG:  received fast shutdown request
db-1     | waiting for server to shut down...2024-02-14 05:29:32.407 UTC [48] LOG:  aborting any active transactions
db-1     | .2024-02-14 05:29:32.409 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
db-1     | 2024-02-14 05:29:32.494 UTC [49] LOG:  shutting down
db-1     | 2024-02-14 05:29:32.494 UTC [49] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-14 05:29:32.503 UTC [49] LOG:  checkpoint complete: wrote 934 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.009 s, sync=0.001 s, total=0.010 s; sync files=0, longest=0.000 s, average=0.000 s; distance=4294 kB, estimate=4294 kB; lsn=0/191C330, redo lsn=0/191C330
db-1     | 2024-02-14 05:29:32.594 UTC [48] LOG:  database system is shut down
db-1     |  done
db-1     | server stopped
db-1     | 
db-1     | PostgreSQL init process complete; ready for start up.
db-1     | 
db-1     | 2024-02-14 05:29:32.797 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-14 05:29:32.797 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1     | 2024-02-14 05:29:32.797 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1     | 2024-02-14 05:29:32.797 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-14 05:29:32.800 UTC [66] LOG:  database system was shut down at 2024-02-14 05:29:32 UTC
db-1     | 2024-02-14 05:29:32.805 UTC [1] LOG:  database system is ready to accept connections
nginx-1 exited with code 0
nginx-1 exited with code 0
api02-1 exited with code 0
api02-1 exited with code 143
api01-1 exited with code 0
api01-1 exited with code 143
db-1     | 2024-02-14 05:33:54.506 UTC [1] LOG:  received fast shutdown request
db-1     | 2024-02-14 05:33:54.506 UTC [1] LOG:  aborting any active transactions
db-1     | 2024-02-14 05:33:54.512 UTC [1] LOG:  background worker "logical replication launcher" (PID 69) exited with exit code 1
db-1     | 2024-02-14 05:33:54.512 UTC [64] LOG:  shutting down
db-1     | 2024-02-14 05:33:54.512 UTC [64] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-14 05:33:54.520 UTC [64] LOG:  checkpoint complete: wrote 908 buffers (5.5%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.007 s, sync=0.001 s, total=0.008 s; sync files=0, longest=0.000 s, average=0.000 s; distance=18848 kB, estimate=18848 kB; lsn=0/2B843B8, redo lsn=0/2B843B8
db-1     | 2024-02-14 05:33:54.598 UTC [1] LOG:  database system is shut down
db-1 exited with code 0
